setup.py

```diff
@@ -84,6 +84,7 @@ def _parse_requirements_file(file_path):
     "onnxruntime>=1.7.0",
     "flask>=1.0.0",
     "flask-cors>=3.0.0",
+    "Pillow>=8.3.2",
```

Comment:
Test dependency

Reply:
will this be auto installed for the Yolo integration now? it is a requirement for from files right?

Reply:
Yes! `pip install deepsparse[yolo]` will install `torchvision` that gives us `Pillow` for free